router/src/main/resources/Dockerfile

```diff
@@ -24,6 +24,8 @@ ARG MG_USER_GROUP=wso2
 ARG MG_USER_GROUP_ID=802
 ARG MG_USER_HOME=/home/${MG_USER}
 
+#COPY maven/mgw-websocket.wasm /usr/local/bin/mgw-websocket.wasm
+#RUN chmod go+r /etc/envoy/envoy.yaml /usr/local/bin/mgw-websocket.wasm
```

Comment:
Why have we commented this here? If this is not needed shall we remove this.

Reply:
fixed